zk-token-sdk/src/zk_token_proof_instruction.rs

```diff
@@ -132,6 +132,27 @@ pub enum ProofInstruction {
     ///   `PubkeyValidityData`
     ///
     VerifyPubkeyValidity,
+
+    /// Verify a 64-bit range proof.
```

Comment:
Regarding the instruction renaming to something like `VerifyRangeProofBitLength64`: I feel like we could maybe deal with in comments, here, instead actually. Currently one needs to dig through the header of zk-token-sdk/src/instruction/range_proof.rs to figure out exactly what this instruction is doing. But this comment block here is going to be the first place anybody who wants to understand what the instruction is goes to. So if we boost up the docs inline here (the other docs are fine too!), then that would also mostly solve my comment (likewise for the instructions being added in the other PR)

Reply:
Yeah, I like the idea of flashing this out some.

Reply:
Great, I will move the instruction descriptions to `zk_token_proof_instruction.rs`.